common/djangoapps/custom_settings/tests/test_handlers.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from course_modes.models import CourseMode
+from custom_settings.models import CustomSettings
+from custom_settings.signals.handlers import initialize_course_settings
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+from openedx.features.philu_utils.pytest_utils import does_not_raise
```

Comment:
utils is duplicated, How about making it simple by renaming `pytest_utils` to `pytest`  as it is understood if it is in `philu_utils` It is utils related to `pytest`

Reply:
Done, to distinguish it from pytest module by the same name, I have renamed `pytest_utils` to `py_test`